src/hotspot/share/opto/idealGraphPrinter.cpp

```diff
@@ -862,24 +904,88 @@ void IdealGraphPrinter::walk_nodes(Node* start, bool edges) {
   }
 }
 
-void IdealGraphPrinter::print_graph(const char* name) {
+static bool skip_frame(const char* name) {
```

Comment:
Nit: the name suggests that this is "skipping a frame". But you are asking if we "should skip the frame". So I would recommend a name like `is_skip_frame`, `must_skip_frame`, `should_skip_frame` or alike.

You could even invert the condition, and make the condition positive: `can_print_stack_frame`.

Totally optional, up to you :)

Reply:
Thanks, I went with `should_skip_frame` (commit 1210180b).